File: engine/model/src/main/scala/net/server/channel/packet/interaction/OmokMovePlayerInteractionPacket.scala
package net.server.channel.packet.interaction

class OmokMovePlayerInteractionPacket(private var _mode: Byte, private var _x: Int, private var _y: Int, private var _type: Int) extends BasePlayerInteractionPacket(_mode = _mode) {
  def x: Int = _x

  def y: Int = _y

  def theType: Int = _type
}

File: engine/engine-model/src/main/java/net/server/channel/packet/interaction/OmokMovePlayerInteractionPacket.java
package net.server.channel.packet.interaction;

public class OmokMovePlayerInteractionPacket extends BasePlayerInteractionPacket {
   private final Integer x;

   private final Integer y;

   private final Integer theType;

   public OmokMovePlayerInteractionPacket(Byte mode, Integer x, Integer y, Integer theType) {
      super(mode);
      this.x = x;
      this.y = y;
      this.theType = theType;
   }

   public Integer x() {
      return x;
   }

   public Integer y() {
      return y;
   }

   public Integer theType() {
      return theType;
   }
}
